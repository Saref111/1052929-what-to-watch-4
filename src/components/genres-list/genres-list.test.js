import React from "react";
import renderer from "react-test-renderer";
import GenresList from "./genres-list.jsx";

const genresList = [`1`, `2`, `3`, `4`];

test(`Snapshot GenreList`, () => {
  const tree = renderer.create(<GenresList
    genresList={genresList}
    currentGenre={`1`}
    onFilterChangeHandler={() => {}}
    films={[]}
    allFilms={[]}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});


