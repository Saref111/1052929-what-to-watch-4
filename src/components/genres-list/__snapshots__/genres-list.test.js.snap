// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot GenreList 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      href="#"
      id="1"
      onClick={[Function]}
    >
      1
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      id="2"
      onClick={[Function]}
    >
      2
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      id="3"
      onClick={[Function]}
    >
      3
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      id="4"
      onClick={[Function]}
    >
      4
    </a>
  </li>
</ul>
`;
